base/applications/mscutils/servman/control.c

```diff
@@ -39,7 +40,8 @@ DoControlService(LPWSTR ServiceName,
             break;
         default:
             /* Unhandled control code */
-            return FALSE;
+           DPRINT1("Unknown control command\n");
+           return FALSE;
```

Comment:
Indentation.
```suggestion
            DPRINT1("Unknown control command\n");
            return FALSE;
```
Print control command value as well?

Reply:
OK